anticipy/forecast_plot.py

```diff
@@ -375,6 +375,10 @@ def plot_forecast(df_fcast, path, output='html', width=None,
             fig = _matplotlib_forecast_create(df_fcast, subplots, sources,
                                               nrows, ncols, width, height,
                                               title, dpi, show_legend)
+            if path is None:
```

Comment:
This doesn't cover the empty string `path=''`. I think it's better to check `if not path:`

Reply:
True. Good catch!